dnf5/commands/remove/remove.cpp

```diff
@@ -61,6 +61,7 @@ void RemoveCommand::configure() {
     auto & context = get_context();
     context.set_load_system_repo(true);
     context.set_load_available_repos(Context::LoadAvailableRepos::NONE);
+    context.base.get_config().get_keepcache_option().set(libdnf5::Option::Priority::RUNTIME, true);
```

Comment:
I have a controversial idea. Please take a seat before continue the reading. What about to move the logic to core behavior. It means in `transaction.cpp` extend the condition `if (!config.get_keepcache_option().get_value()) {` by `OR` transaction does not have any inbound package. I believe that kind of information is available to that part of the code. This behavior will unify behavior with daemon and other API users. Please take it as subject for a discussion.